Sources/KituraContracts/CodableQuery/Coder.swift

```diff
@@ -32,6 +32,10 @@ public class Coder {
 
     /// Helper method to extract the field name from a CodingKey array
     public static func getFieldName(from codingPath: [CodingKey]) -> String {
-        return codingPath.flatMap({"\($0)"}).joined(separator: ".")
+		#if swift(>=4.1) || os(Linux)
```

Comment:
This doesn't make sense - it changes behaviour for Swift 4 on Linux?

Reply:
Amended this now, there were failing Linux travis builds and thought it was to do with that. It wasn't. Resolved that now.